testutils/source.go

```diff
@@ -1807,7 +1838,8 @@ func main() {
 		panic(err)
 	}
 	fmt.Println(result)
-}`}, 1, gosec.NewConfig()}, {[]string{`
+}`}, 1, gosec.NewConfig()},
```

Comment:
These are results of `gofumpting` the file. Am I missing some config, or did `gofumpt` get more litigious?